src/main.d

```diff
@@ -365,6 +365,12 @@ int main(string[] args)
 				log.vlog("[M] Item deleted: ", path);
 				try {
 					sync.deleteByPath(path);
+				} catch(SyncException e) {
+					if (e.msg == "The item to delete is not in the local database") {
+						log.vlog("Item cannot be deleted because not found in database");
+					} else {
+						log.logAndNotify(e.msg);
```

Comment:
I changed this `log` to `logAndNotify`, seems natural here. My original patch was based on master before the merge of notifications.